starter-code/index.html

```diff
@@ -2,41 +2,86 @@
 <html lang="en">
 <head>
   <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <link rel="stylesheet" href="css/main.css">
   <title>Spotify</title>
 </head>
+
 <body>
-Premium Discover Help Download
-
-Music for everyone.
-Spotify is now free on mobile, tablet and computer.
-Listen to the right music, wherever you are.
-
-What’s on Spotify?
-
-Millions of Songs
-There are millions of
-songs on Spotify
-
-HD Music
-Listen to music as if you
-were listening live
-
-Stream Everywhere
-Stream music on your
-smartphone, tablet or
-computer
-
-It’s as yeezy as Kanye West.
-Search
-Know what you want to listen to?
-Just search and hit play.
-Browse
-Check out the latest charts,
-brand new releases and great
-playlists for right now.
-Discover
-Enjoy new music every Monday
-with your own personal playlist.
-Or sit back and enjoy Radio.
+  <nav id="nav-bar" class="container">
+    
+    <img src="images/spotify-logo.png">
+    
+    <ul>
+      <a href="#" ><il>Premium</il></a>
+      <a href="#"><il>Discover</il></a>
+      <a href="#"><il>Help</il> </a>
+      <a href="#"><il>Download</il></a>  
+  </ul>
```

Comment:
Under a ```ul``` tag you should use ```li``` tags as siblings and not wrap them around other elements. You used ```il``` which is an invalid tag so be careful.